### PR TITLE
[ICE] coturn: switch to lt-cred-mech authorization

### PR DESCRIPTION
**Pull request by Geosearchef**

### CODE REVIEW

File: config.template/faf-coturn/faf-coturn.env [hunk 1]
```diff
@@ -1,2 +1,6 @@
-EXTERNAL_IP=127.0.0.1
-SKIP_AUTO_IP=true
+TURN_PORT=3478
+TURN_PORT_START=10000
+TURN_PORT_END=20000
+TURN_SECRET=banana
+TURN_SERVER_NAME=faf-coturn
+TURN_REALM=faforever.com
```
Comment (Brutus5000) [inline config.template/faf-coturn/faf-coturn.env:6]: The template should always be usable from localhost. Is this realm somehow checked or used to connect to?

File: docker-compose.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ services:
   #
   faf-coturn:
     container_name: faf-coturn
-    image: bprodoehl/turnserver
+    image: geosearchef/coturn-lt-cred
```
Comment (Brutus5000) [inline docker-compose.yml]: I don't know what this image contains, but shouldn't we move it to the faforever repo?

File: config.template/faf-coturn/faf-coturn.env (hunk 1 above)
Comment (Geosearchef) [reply config.template/faf-coturn/faf-coturn.env:6]: the realm is the realm in which the user account has to exist, if the user doesn't specify a realm, the server will reply with an error and its realm so the user can retry using this realm, we don't use realms, so it really doesn't matter what is specified there, but it needs to be specified to use lt-cred-mech

File: docker-compose.yml (hunk 2 above)
Comment (Geosearchef) [reply docker-compose.yml]: we should (before it wasn't better :P , actually this image was terrible), I'm gonna move that to the faforever git and docker hub repos

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**